src/main/kotlin/apply/application/AdministratorDtos.kt

```diff
@@ -0,0 +1,38 @@
+package apply.application
+
+import apply.domain.administrator.Administrator
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Size
+
+data class AdministratorData(
+    @field:Size(min = 1, max = 20)
+    @field:Pattern(
+        regexp = "[0-9a-zA-Z가-힣]+",
+        message = "이름은 영어, 한글, 숫자만 가능합니다."
+    )
```

Comment:
c: `RegisterUserRequest`의 `name`을 참고하면 어떨까요?

```suggestion
    @field:Pattern(regexp = "[가-힣]{1,30}", message = "올바른 형식의 이름이어야 합니다")
```